clients/forms-data-management/src/FormDataManagementClient.ts

```diff
@@ -399,6 +399,7 @@ export class FormDataManagementClient extends WsgClient {
   "imjs_form_data_management_relying_party_uri";
   public constructor() {
     super("sv1.2");
+    // this.baseUrl = "https://api.bentley.com/formswsg";
```

Comment:
Any ETA on this service being available through APIM?

Reply:
It is available in APIM but the integration tests are failing when using APIM endpoints (I wrote a status email about that last year). I'm not a maintainer of Forms Service so I can't tell when it will be fixed.

Reply:
Okay, sounds good.  I'll follow up with everyone internally and not mark this full item complete until everything is covered.